tests/acceptance/test_simplified_long_covid_study.py

```diff
@@ -17,7 +17,6 @@
 from ..lib.util import extract
```

Comment:
This file should probably go - I don't think it gives us anything useful over the full long covid one, it's just a subset of that.

Reply:
Addressed in 73f2b1c21f18d5d9b7ec5e814b2bc65025d3d5d6.